libs/api/domains/cms/src/lib/cms.contentful.service.ts

```diff
@@ -568,6 +569,20 @@ export class CmsContentfulService {
     return result.items.map(mapHomepage)[0]
   }
 
+  async getFrontpage({ lang }: { lang: string }): Promise<Frontpage> {
+    const params = {
+      ['content_type']: 'frontpage',
+      include: 10,
+      order: '-sys.createdAt',
+    }
+
+    const result = await this.contentfulRepository
+      .getLocalizedEntries<types.IFrontpageFields>(lang, params)
+      .catch(errorHandler('getFrontpage'))
+
+    return result.items.map(mapFrontpage)[0]
```

Comment:
Maybe we should add a page identifier field like we do on the "Generic Overview Page" that is more robust

Reply:
May I ask why?
What does that add?